src/libasr/utils.h

```diff
@@ -38,6 +38,7 @@ struct CompilerOptions {
     bool implicit_interface = false;
     std::string target = "";
     std::string arg_o = "";
+    bool arg_g = false;
```

Comment:
I would call it `emit_debug_info` or something like that. It can be done in a subsequent PR.